cinema/serializers.py

```diff
@@ -69,6 +87,7 @@
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = MovieImageSerializer()
```

Comment:
The `movie_image` field in `MovieSessionListSerializer` should instantiate `MovieImageSerializer`. Change `movie_image = MovieImageSerializer` to `movie_image = MovieImageSerializer()`.